### ISSUE TITLE
Unsubscribing from emails we send to users does not work

### ISSUE THREAD

**Issue by gtsiolis** [gitpod-io/gitpod#14755]
### Bug description

Clicking on **unsubscribe here** on emails we send to users does not seem to work.

Users see a successful unsubscription message, but opening /notifications does not seem to reflect the change.

Cc @chrifro @MichaelAring @phimae because of a [relevant task](https://www.notion.so/gitpod/7431896ad7c04b7a8ad45af2632d41c6) (internal)

| Successful unsubscription message | Notifications page |
|-|-|
| <img width="1440" alt="Screenshot 2022-11-16 at 7 16 12 PM (2)" src="https://user-images.githubusercontent.com/120486/202248989-b51f43fe-505e-46b0-a58b-e1fcd92a7a23.png"> | <img width="1440" alt="Screenshot 2022-11-16 at 7 16 21 PM (2)" src="https://user-images.githubusercontent.com/120486/202249335-30bae033-35ce-4e97-9495-65f81026377f.png"> |

### Steps to reproduce

Click on **unsubscribe here** on an email we sent like the one titled "_Personalize your workspace – ..._"

**Comment by chrifro** [comment]
Related context: [internal discussion](https://gitpod.slack.com/archives/C022DFDLAQL/p1649401546492159)

I just tested the flow for the Changelog email and the unsubscribe flow worked fine for me. Could you check if the issue persists for you @gtsiolis? Which unsubscribe link did you use?

**Comment by gtsiolis** [comment]
@chrifro This remains an issue for emails outside the defined categories in `/notifications` like changelog, etc.

For example, I just received an email about the _latest case study_. Clicking unsubscribe redirects users to that successful unsubscription message shown in the issue description with a link to `/notifications`, which does not make any changes to the `/notifications`.

---

This is [what I tried to describe](https://www.notion.so/Customer-io-v2-7431896ad7c04b7a8ad45af2632d41c6?d=02caef61ea1a461da399ef65bf91e27a#f840c684e34f4de5a4913ef4c4e7a0c3) (internal) as a root problem in the relevant task linked in the issue description.

> @gtsiolis:  Users may find this quite frustrating as their options do not reflect what’s happening and that we’re abusing their preferences this way. ❗

> @gtsiolis: That partially explains also why we’re receiving so much complains about sending too many emails in the past. I learned only today that we’re using these opt-ins to send users emails besides the Onboarding guide, Changelog, and Developer Experience & Product Tips. 💡

---

Also, from another [relevant discussion](https://gitpod.slack.com/archives/C020VCB0U5A/p1677063850262589?thread_ts=1677062507.271759&cid=C020VCB0U5A) (internal):

> @gtsiolis: IMO we've been somewhat aggressive with adding users to email buckets in the past.

> 1. Misleading information where users can not confirm if they actually unsubscribed, see [#14755](https://github.com/gitpod-io/gitpod/issues/14755).
> 2. Unnecessary notification settings in the dashboard, see [#14746](https://github.com/gitpod-io/gitpod/pull/14746).
> 3. Missing docs on what kind of notifications users get, see [#3038](https://github.com/gitpod-io/website/pull/3038).
> 4. Sometimes emails we sent contains broken links or obsolete content, see [#14754](https://github.com/gitpod-io/gitpod/issues/14754).
> 5. ...

---

If you think this is not a problem we need to solve right now, let's close this issue.

**Comment by chrifro** [comment]
>For example, I just received an email about the latest case study. Clicking unsubscribe redirects users to that successful unsubscription message shown in the issue description with a link to /notifications, which does not make any changes to the /notifications.

I agree that's not a great flow, we are aware of that and raised the concern [here](https://gitpod.slack.com/archives/C046Z4BBP2N/p1678197993761979?thread_ts=1678189317.044429&cid=C046Z4BBP2N) (internally) as well. 

cc @MichaelAring as you set up the email broadcast
